src/containers/quiz/quiz-question/Question.tsx

```diff
@@ -0,0 +1,158 @@
+import { ChangeEventHandler, FC } from 'react'
+import { useTranslation } from 'react-i18next'
+import { Box, Typography, FormGroup, RadioGroup, SxProps } from '@mui/material'
+import { questionType } from '~/components/question-editor/QuestionEditor.constants'
+import AppCard from '~/components/app-card/AppCard'
+import CheckIcon from '@mui/icons-material/Check'
+import CloseIcon from '@mui/icons-material/Close'
+
+import { Question } from '~/types'
+import { styles } from './Question.styles'
+import { isCorrectAnswer } from './Question.constants'
+import Answer from '../question-answer/Answer'
```

Comment:
also here, use alias

Reply:
Thnaks for noticing, used allias here and in other places